### ISSUE TITLE
Problem when executing on Windows 11 23H2

### ISSUE THREAD

**Issue by HotCakeX** [x0xr00t/Automated-MUlti-UAC-Bypass#28]
OS Build: 22631.3007 

screenshots

<br>

<img width="1280" alt="image" src="https://github.com/x0xr00t/Automated-MUlti-UAC-Bypass/assets/118815227/cc3f64eb-b5db-4d5f-93f0-3cca748bff72">

<img width="1280" alt="image" src="https://github.com/x0xr00t/Automated-MUlti-UAC-Bypass/assets/118815227/5f868a25-18ea-4cd6-b2b7-de1f8e184c66">

<img width="1280" alt="image" src="https://github.com/x0xr00t/Automated-MUlti-UAC-Bypass/assets/118815227/44040486-0b61-4213-aed4-8f325706c4ad">

**Comment by x0xr00t** [comment]
@HotCakeX take a look at the errors. 
1.) directory exist. (change line mockfolder to "C:\Windows \system32"
2.) file exist, (delete the sl0p.dll in either "C:\Windows\" and make sure its placed the sl0p.dll in the mock folder C:\Windows \system32\ ) 
3.) make sure powershell v2 or higher is present on the system and not v1

**Comment by HotCakeX** [comment]
@x0xr00t I did what you said but still not working, I always get the UAC prompt.

**Comment by x0xr00t** [comment]
@HotCakeX  so the uac does not auto bypass prompt ? i will look in to that

**Comment by HotCakeX** [comment]
@x0xr00t no, when I get the uac prompt i select no (since the idea is to bypass it automatically instead of accepting it) so its not bypassed. thank you

**Comment by x0xr00t** [comment]
@HotCakeX After alot of invest, i can conclude its not the code, that has a issue!. 
The UAC popup been bypassed, the popup you get is the one for exec of powershell. 
Which has nothing to do with the UAC prompt, their for the UAC prompt is bypassed. 

What u need to do is the following, read section setup on the readme.MD. 
It explains that u need to use the command there, in order to get rid of that prompt. 
For that i listed a command u need to exec, for powershell before running the .ps1 script. 

I will explain: the windows first will prompt a UAC prompt asking if u want to exec this, then after that it prompts a powershell prompt to ask if u want to run the ps code. The issues u encounter has to do with the second prompt which is the powershell prompt to exec ps code. Their for the claim that its not working with the UAC prompt is bogus. As it bypasses the UAC prompt which in the process is the first prompt of the 2 prompts, that would appear.

To solve your issue, Read the setup part of the readme.MD
Setup

    Change username in the script (where ever needed)
    Set-ExecutionPolicy -ExecutionPolicy {Unrestricted or Bypass} -Scope CurrentUser
    Or use one of the bypasses like type file.ps1 | poweshell.exe -no-profile or what ever suites
    Add a automation process to disable tamper once uac been invoked (this can be done!!)

What u need to do is use the following.
* Set-ExecutionPolicy -ExecutionPolicy Bypass -Scope CurrentUser Or use 
* Set-ExecutionPolicy -ExecutionPolicy Unrestricted -Scope CurrentUser

With that ur issues would be solved. 
Kinds, 

x0xr00t

**Comment by HotCakeX** [comment]
Of course I changed the execution policy, that's how the script was able to even run.

The script gets the username automatically

```powershell
$user = $(cmd.exe /c echo %username%)
```

I'm not sure what you mean by changing it wherever needed ? can you be more specific ?

it's clearly not working. I've even added Downloads folder to the exclusion list so MS Defender is not stopping it. Running on a non-hardened Windows OS.

<img width="1280" alt="image" src="https://github.com/x0xr00t/Automated-MUlti-UAC-Bypass/assets/118815227/39bf6a48-4ccd-4c92-a99b-8824258f734b">

<img width="1280" alt="image" src="https://github.com/x0xr00t/Automated-MUlti-UAC-Bypass/assets/118815227/3c9225cb-a9d0-4464-b4bd-3d5099564533">

<img width="1280" alt="image" src="https://github.com/x0xr00t/Automated-MUlti-UAC-Bypass/assets/118815227/180b04c3-8d83-4db9-9263-74ea876efbb9">

<br>
<br>

Even the insecure PS v2 is installed

<img width="344" alt="image" src="https://github.com/x0xr00t/Automated-MUlti-UAC-Bypass/assets/118815227/82530e8d-9d7c-40e5-905b-98608ac70522">

**Comment by x0xr00t** [comment]
@HotCakeX What u likely need to do is the following. 
Open a terminal in powershell then fetch the execution path of powershell, either v2.0 or the v7 one. Best would be the v7 one, but u could also fetch the v2 location. Then edit the .cs file with the location of the ps execution path, lets say u use powershell v7 use that path, and replace it with the string in the .cs file. Remove the dll and remake the dll. Then re-run the code .ps1 

should work after u done that. 
~x0xr00t~
